xc7/make/arch_define.cmake

```diff
@@ -42,7 +42,7 @@ function(ADD_XC7_ARCH_DEFINE)
     BIN_EXTENSION bit
     FASM_TO_BIT ${PRJXRAY_DIR}/utils/fasm2frames.py
     FASM_TO_BIT_CMD "${CMAKE_COMMAND} -E env \
-    PYTHONPATH=${PRJXRAY_DIR}:${PRJXRAY_DIR}/third_party/fasm \
+    PYTHONPATH=${symbiflow-arch-defs_BINARY_DIR}/env/conda/lib/python3.7/site-packages:${PRJXRAY_DIR}:${PRJXRAY_DIR}/third_party/fasm \
```

Comment:
This is wrong and indicates your PYTHON3 variable is incorrect.

Reply:
Yes, I thought the same, the problem is that I cannot find a way let it correctly compile.

Reply:
Do you happen to have an env var PYTHON3?  The symbiflow CMake allows executable overrides via an env var of the name.